src/scenic/simulators/webots/model.scenic

```diff
@@ -87,12 +113,20 @@ class Ground(WebotsObject):
     """
 
     allowCollisions: True
-    webotsName: 'Ground'
-    positionOffset: (-self.width/2, -self.length/2)  # origin of ElevationGrid is at a corner
+    webotsName: 'GROUND'
```

Comment:
Why change this name and gratuitously break existing scenarios?

Reply:
As far as I can tell Webots DEF statements are supposed to use all caps and underscores. I changed this to match that convention, since people will probably expect  that (or should to match Webots style) and it really shouldn't have been "Ground" by default in the first place. I can definitely fix the docstring though.

Reply:
OK.